misc/python/materialize/scalability/result_analyzers.py

```diff
@@ -0,0 +1,122 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+
+import pandas as pd
+
+from materialize.scalability.endpoint import Endpoint
+from materialize.scalability.regression import Regression, RegressionOutcome
+from materialize.scalability.result_analyzer import (
+    ResultAnalyzer,
+)
+from materialize.scalability.workload_result import WorkloadResult
+
+COL_CONCURRENCY = "concurrency"
+COL_COUNT = "count"
+COL_TPS = "tps"
+COL_TPS_DIFF = "tps_diff"
+COL_TPS_DIFF_PERC = "tps_diff_perc"
+COL_TPS_BASELINE = "tps_x"
+COL_TPS_OTHER = "tps_y"
+
+
+class DefaultResultAnalyzer(ResultAnalyzer):
+    def __init__(self, max_deviation_in_percent: float):
+        self.max_deviation_in_percent = max_deviation_in_percent
+
+    def determine_regressions_in_workload(
+        self,
+        regression_outcome: RegressionOutcome,
+        regression_baseline_endpoint: Endpoint,
```

Comment:
just `baseline_endpoint` here and elsewhere, as this is a mouthful otherwise.

Reply:
✅